integration/dataservice-hosting-tests/tests-integration/tests/src/test/java/org/wso2/ei/dataservice/integration/test/jira/issues/DS1103DuplicateDataServiceTestCase.java

```diff
@@ -32,25 +33,25 @@
  * This test was written to verify the fix for https://wso2.org/jira/browse/DS-1103.
  */
 public class DS1103DuplicateDataServiceTestCase extends DSSIntegrationTest {
-	private final String serviceName = "H2SimpleJsonTest";
-	private List<File> sqlFileLis;
+	private final String serviceName = "DuplicateDataServiceTest";
+	private List<File> sqlFileList;
 
 	@BeforeClass(alwaysRun = true)
 	public void serviceDeployment() throws Exception {
 		super.init();
-		sqlFileLis = new ArrayList<>();
-		sqlFileLis.add(selectSqlFile("CreateTables.sql"));
-		sqlFileLis.add(selectSqlFile("Students.sql"));
-		deployService(serviceName,
-		              createArtifact(getResourceLocation() + File.separator + "dbs" + File.separator + "rdbms" +
-		                             File.separator + "h2" + File.separator + serviceName + ".dbs", sqlFileLis));
+		sqlFileList = new ArrayList<>();
+		sqlFileList.add(selectSqlFile("CreateTables.sql"));
+		sqlFileList.add(selectSqlFile("Students.sql"));
+		SqlDataSourceUtil dataSource = new SqlDataSourceUtil(sessionCookie
+				, dssContext.getContextUrls().getBackEndUrl());
```

Comment:
Move the , to just after the sessionCookie in the previous line?

Reply:
Done